go.mod

```diff
@@ -4,17 +4,27 @@ go 1.16
 
 require (
 	cloud.google.com/go v0.82.0
+	github.com/DataDog/zstd v1.5.0 // indirect
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
+	github.com/cockroachdb/errors v1.8.6 // indirect
+	github.com/cockroachdb/logtags v0.0.0-20211118104740-dabe8e521a4f // indirect
+	github.com/cockroachdb/pebble v0.0.0-20211208200752-b10ca3a5c8f1 // indirect
+	github.com/cockroachdb/redact v1.1.3 // indirect
```

Comment:
I believe most of these dependencies are added by the Honu pebble dependency (and the ones below).